validation.lua

```diff
@@ -70,6 +70,11 @@ users_match = function (self)
     return (self.session.username == self.params.username)
 end
 
+user_is_admin = function (self)
+    local user = Users:find(self.session.username)
+    return user.isadmin
```

Comment:
will throw NPE if user does not exist

Reply:
also might want to have `assert_admin` share a code path with this function to keep things consistent

Reply:
oh makes sense. I guess that's one reason for doing a `current_user` thing...makes it easy to have an `assert_user_is_logged_in` or something.